app/templates/_exchanges.html

```diff
@@ -0,0 +1,290 @@
+
+<!--
+# exchanges logic
+# loop through exchanges
+#   find ones that have this user's hospital id somewhere in the linked exchange -> grab whole "exchanges"
+#   exchange id -> exchanges.id
+#   when created -> exchanges.creation_timestamp
+#   when updated -> exchanges.updated_timestamp
+#   loop through exchanges
+#     print out exchange
+#     print out status
+#     depending on status, show optional button
+
+# The final table looks like:
+# Exchange ID   when created    when updated    exchange part 1     status part 1: optional button (verify/shipped/received)
+#                                               exchange part 2     status part 2: optional button
+#                                               ...                 ...
+-->
+
+{% macro whole_exchange_td(loop, ex) %}
```

Comment:
Macro which is used for a table cell that spans multiple rows for an entire `Exchanges` (ex. `ID`, `Created`).